generated/nixnet/nixnet_service.cpp

```diff
@@ -169,7 +169,7 @@ namespace nixnet_grpc {
       auto session_ref_grpc_session = request->session_ref();
       nxSessionRef_t session_ref = session_repository_->access_session(session_ref_grpc_session.id(), session_ref_grpc_session.name());
       u8* value_buffer = (u8*)request->value_buffer().c_str();
-      u32 size_of_value_buffer = static_cast<u32>(request->value_buffer().size());
+      u32 size_of_value_buffer = static_cast<u32>(request->value_buffer().size() * sizeof(u8));
```

Comment:
note: This change to multiply buffer size with size of data type for params expecting "len-in-bytes" was introduced in earlier PR. The autogenerated change was introduced in this PR because of bad merge in previous PR.
Same is mentioned in Rohan's [PR](https://github.com/ni/grpc-device/pull/622)